sktime/forecasting/base/_meta.py

```diff
@@ -25,10 +26,14 @@ class _HeterogenousEnsembleForecaster(_HeterogenousMetaEstimator, BaseForecaster
     # this must be an iterable of (name: str, estimator, ...) tuples for the default
     _steps_fitted_attr = "forecasters_"
 
-    def __init__(self, forecasters, n_jobs=None, fc_alt=None):
+    def __init__(
+        self, forecasters, backend=None, backend_params=None, n_jobs=None, fc_alt=None
+    ):
         if forecasters is not None:
             self.forecasters = forecasters
-        self.n_jobs = n_jobs
+        self.backend = None
+        self.backend_params = None
+        self.n_jobs = n_jobs  # Retained for backward compatibility
```

Comment:
the parameter is no longer used. If used, it should default to `joblib`, the previous defualt.

If not too difficult, I would also suggest to add deprecation.